systemvm/debian/opt/cloud/templates/conntrackd.conf.templ

```diff
@@ -413,6 +413,9 @@ General {
         #   ESTABLISHED CLOSED TIME_WAIT CLOSE_WAIT for TCP
         # }
     }
+    Stats {
```

Comment:
@shawniverson 
should this be a separated section or under the "General" section ?

Reply:
It looks like there are no options in Sync to just disable logging there.  I was trying to avoid turning off logging in the General block.  It seems to be the only option.

Reply:
I will need some advice from CloudStack developers on this. I can revise this PR or submit a new PR with logging off in in the General block instead, but I need feedback on whether any of the conntrackd logging is even being used before proceeding.

Reply:
from the man page:
```
[LogFile <on|off|filename>](https://manpages.debian.org/testing/conntrackd/conntrackd.conf.5.en.html#LogFile)
    Enable [conntrackd(8)](https://manpages.debian.org/testing/conntrackd/conntrackd.8.en.html) to log to a file.

    Example: LogFile on

    Default is off. The default logfile is /var/log/conntrackd.log.

[Syslog <on|off|facility>](https://manpages.debian.org/testing/conntrackd/conntrackd.conf.5.en.html#Syslog)
    Enable connection logging via Syslog. If you set the facility, use the same as in the Stats section, otherwise you'll get a warning message.

    Example: Syslog local0

    Default is off.
 ```

Reply:
@shawniverson there was talk of creating a central logging facility once. For that the syslog facility would be nice.
If you turn off logging I would say it should be done with a global setting. I am not sure if the logging of conntrackd is used functionally but I imagine that it is useful for people running redundant routers.

Reply:
@DaanHoogland A global setting makes sense. I'll study this and see what I can do.